src/lib.nr

```diff
@@ -89,9 +273,10 @@ pub mod sha512 {
 
 }
 
+// methods here are interface-compatible with stdlib
```

Comment:
sha256 and sha512 have been removed from the stdlib so we don't need to maintain compatibility btw.

Reply:
what about the sha256 blackbox interface? that's the one I want to maintain compatibility with tbh